bin/container.sh

```diff
@@ -228,6 +228,19 @@ EOF
   docker ps 2>/dev/null
 }
 
+exec_commands_simple() {
+        prefix=$1
```

Comment:
The rest of the functions have only two spaces as indentation. Also applies for [bin/wasabi.sh:314](https://github.com/intuit/wasabi/pull/89/files#diff-5586624b459b5bffc7ce7afc0f554717R314).